serveradmin/apps/models.py

```diff
@@ -34,6 +34,7 @@ class Application(models.Model):
     owner = models.ForeignKey(User, on_delete=models.CASCADE)
     location = models.CharField(max_length=150)
     disabled = models.BooleanField(default=False)
+    last_login = models.DateTimeField(null=True, default=None, editable=False)
```

Comment:
Help me here quick, what does `editable=False` do again?

Reply:
The field becomes non-editable in the autogenerated admin ui.

Reply:
Nice. Thx. ^^